server/models/Availablity.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");

```

Comment:
Can't see where you used this. You can remove it if not used.

Reply:
Yeah sure forgot to remove them when copy from user model